src/Classes/Item.lua

```diff
@@ -63,6 +63,198 @@ function ItemClass:ResetInfluence()
 	end
 end
 
+-- Special function to store unique instances of modifier on specific item slots
+-- that require special handling for ItemConditions. Only called if line #102 is
+-- uncommented
+local specialModifierFoundList = {}
+local inverseModifierFoundList = {}
+function getTagBasedModifiers(tagName, itemSlotName)
```

Comment:
Since this is only loaded on update, I think it should be in the Export scripts.  Then in `Data` we can load the completed table

Reply:
made it local but left it as is as this function never gets called and is specific to this file only